FlowCrypt/Controllers/Settings/Settings List/SettingsViewController.swift

```diff
@@ -31,17 +31,35 @@ final class SettingsViewController: TableNodeViewController {
             case .experimental: return "settings_screen_experimental".localized
             }
         }
+
+        static func allCases(with rules: OrganisationalRules?) -> [Settings] {
+            guard let rules = rules else {
+                return allCases
+            }
+            let cases: [Settings]
+            if !rules.canBackupKeys {
+                cases = [.privacy, .contacts, .keys, .atteseter, .notifications, .legal, .experimental]
```

Comment:
This addresses the issue as written, but it will get messy quickly once we add more OrgRules that restrict some other items. Let's say we have three such rules, so that's 9 combinations, then you'll need to declare exact lists for each one of them. Imagine then later you want to add a menu item that's unrestricted - you'll have to add it in 9 places. Already now you would need to add it in two places - error prone.

Instead of this approach, do the following: start with a full list, then use the OrgRules to modify the list one by one. For now we have one OrgRule, so something like:

```
// pseudo code
let menuItems = allMenuItems.copy()
if !rules.canBackupKeys {
  menuItems.remove(.backups)
}
return menuItems
```
Then it's trivial to add more rules after that, and trivial to add more items etc.

Reply:
done